controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -428,18 +430,26 @@ func (r *Reconciler) processBatchAttachmentAndFilterVolumeSpecs(
 
 		pvc, err := r.getPVC(ctx, vol.PersistentVolumeClaim.ClaimName)
 		if err != nil {
-			retErr = errOrNoRequeueErr(retErr, err)
+			retErr = errOrNoRequeueErr(retErr,
+				fmt.Errorf("failed to get pvc. PVC name: %s, Vol name: %s, error: %w",
+					pvc.Name, vol.Name, err))
 			continue
 		}
 
 		// Handle PVC with WFFC first since it handles PVCS that are unbound.
 		if err := r.handlePVCWithWFFC(ctx, vol, pvc); err != nil {
-			retErr = errOrNoRequeueErr(retErr, err)
+			retErr = errOrNoRequeueErr(retErr,
+				fmt.Errorf("failed to handle PVC with WFFC. PVC name: %s, Vol name: %s, error: %w",
```

Comment:
Similar message change too. Also, with this error having the PVC name, the name can be removed from some of the errors returned by handlePVCWithWFFC